docs/framework/guides/creating-simple-plugin-timestamp.md

```diff
@@ -180,7 +215,12 @@ class Timestamp extends Plugin {
 		const editor = this.editor;
 
 		editor.ui.componentFactory.add( 'timestamp', () => {
-			//...
+			const button = new ButtonView();
+
+            button.set( {
+                label: 'Timestamp',
+                withText: true
+            } );
```

Comment:
See my previous comment. I'm not sure if in this case adding the full snippet is a good idea, as I have a feeling of breaking the flow and distracting - "should I change something in my `ButtonView` definition too?". If we have `// ...` for the unchanged part of the code, we make it clear that now we work on a new part of the code and the previous one remains unchanged.

Reply:
It was fixed as agreed by using an ellipse with the comment.